sway/desktop/fx_renderer/fx_renderer.c

```diff
@@ -756,21 +794,10 @@ void fx_render_box_shadow(struct fx_renderer *renderer, const struct wlr_box *bo
 	inner_box.width -= blur_sigma * 2;
 	inner_box.height -= blur_sigma * 2;
 
-	glEnable(GL_STENCIL_TEST);
```

Comment:
should we call fx_renderer_stencil_enable() here?

Reply:
Removed that function due to it being unused and actually unneeded due to `fx_renderer_stencil_mask_init` doing the same thing :)